frontend/src/container/LiveLogs/LiveLogsListChart/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { LIVE_TAIL_GRAPH_INTERVAL } from 'constants/liveTail';
+import { PANEL_TYPES } from 'constants/queryBuilder';
+import LogsExplorerChart from 'container/LogsExplorerChart';
+import { useGetExplorerQueryRange } from 'hooks/queryBuilder/useGetExplorerQueryRange';
+import { useQueryBuilder } from 'hooks/queryBuilder/useQueryBuilder';
+import { useEventSource } from 'providers/EventSource';
+import { useMemo } from 'react';
+import { Query } from 'types/api/queryBuilder/queryBuilderData';
+import { QueryData } from 'types/api/widgets/getQuery';
+import { DataSource, LogsAggregatorOperator } from 'types/common/queryBuilder';
+
+function LiveLogsListChart(): JSX.Element {
+	const { stagedQuery } = useQueryBuilder();
+	const { isConnectionOpen, isConnectionLoading } = useEventSource();
+
+	const listChartQuery: Query | null = useMemo(() => {
+		if (!stagedQuery) return null;
+
+		return {
+			...stagedQuery,
+			builder: {
+				...stagedQuery.builder,
+				queryData: stagedQuery.builder.queryData.map((item) => ({
+					...item,
+					disabled: false,
+					aggregateOperator: LogsAggregatorOperator.COUNT,
+					filters: {
+						...item.filters,
+						items: item.filters.items.filter((item) => item.key?.key !== 'id'),
+					},
+				})),
+			},
+		};
+	}, [stagedQuery]);
+
+	const { data, isFetching } = useGetExplorerQueryRange(
+		listChartQuery,
+		PANEL_TYPES.TIME_SERIES,
+
+		{
+			enabled: isConnectionOpen,
+			refetchInterval: LIVE_TAIL_GRAPH_INTERVAL,
+		},
+		{ dataSource: DataSource.LOGS },
+	);
+
+	const chartData: QueryData[] = useMemo(() => {
```

Comment:
I think useMemo isn't required here. We aren't computing anything

Reply:
@YounixM We use it like props with memo component. Also, we have context which causes rerender, so useMemo is a good solution here and below examples. If we don't have computing, it doesn't mean we shouldn't use useMemo.